metals/src/main/scala/scala/meta/internal/metals/watcher/FileWatcherEvent.scala

```diff
@@ -14,6 +14,7 @@ object FileWatcherEvent {
     case object Create extends EventType
```

Comment:
TODO merge Create and Modify events, see https://github.com/scalameta/metals/pull/3758#issuecomment-1080005296